developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_30.rst

```diff
@@ -145,12 +145,43 @@ In this release support for PHP 8.0 was removed. Follow the steps below to make
 
   {
     "require": {
-      "php": ">=8.1 <=8.3"
+      "php": ">=8.1 <=8.4"
     }
   }
 
 3. If you have :ref:`continuous integration <app-ci>` set up, remove PHP 8.0 from the matrices of tests and linters.
 
+Support for PHP 8.4 added
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 30.0.2
+
+In this release support for PHP 8.4 was added. Follow the steps below to make your app compatible.
+
+1. If ``appinfo/info.xml`` has a dependency specification for PHP, increase the ``max-version`` to 8.4.
+
+.. code-block:: xml
+
+  <dependencies>
+    <php min-version="8.1" max-version="8.4" />
+    <nextcloud min-version="26" max-version="28" />
```

Comment:
```suggestion
    <nextcloud min-version="26" max-version="30" />
```
Makes no sense to support 8.4 but not 30.

Reply:
Sorry, was copy pasta from adding 8.3 and seems I didnt replace all.
```suggestion
    <nextcloud min-version="28" max-version="30" />
```